e2e/cypress/support/ldap_server_commands.js

```diff
@@ -0,0 +1,7 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+Cypress.Commands.add('addLDAPUsers', () => {
+    cy.exec('ldapmodify -x -D "cn=admin,dc=mm,dc=test,dc=com" -w mostest -H ldap:// -f cypress/fixtures/test-data.ldif -c', {failOnNonZeroExit: false});
```

Comment:
- For other LDAP tests, we're using [tests/test-data.ldif](https://github.com/mattermost/mattermost-server/blob/master/tests/test-data.ldif). I wonder if there's side-effect when modifying another set of `cypress/fixtures/test-data.ldif` users.
- Please consider the host of LDAP server at [cypress.json](https://github.com/mattermost/mattermost-webapp/blob/master/e2e/cypress.json#L19-L20) since in CI, Cypress agent is running in a different host than the LDAP server.

Reply:
So, I created these in a separate base dn 
dn: ou=e2etest,dc=mm,dc=test,dc=com (fixtures/test-data.ldif)
dn: ou=testusers,dc=mm,dc=test,dc=com (tests/test-data.ldif)

I wanted to keep them completely separate from other test data.